src/internal/operations/test/helper_test.go

```diff
@@ -398,6 +398,21 @@ func generateContainerOfItems(
 	restoreCfg.Location = destFldr
 	restoreCfg.IncludePermissions = true
 
+	if sel.Service == selectors.ServiceGroups {
+		restoreCfg.SubService.Type = path.SharePointService
+		restoreCfg.SubService.ID = siteID
+	}
+
+	var protectedResource idname.Provider = sel
+
+	// In case of  groups, we use the root site as the restore target
```

Comment:
Do we have to make it configurable by caller? I didn't think it was required as this is test code.